lib/matplotlib/pyplot.py

```diff
@@ -2650,14 +2650,14 @@ def pie(
         pctdistance=0.6, shadow=False, labeldistance=1.1,
         startangle=None, radius=None, counterclock=True,
         wedgeprops=None, textprops=None, center=(0, 0), frame=False,
-        rotatelabels=False, *, data=None):
+        rotatelabels=False, normalize=None, *, data=None):
```

Comment:
~~~
        rotatelabels=False, normalize=None, *, data=None):
~~~
your boilerplate.py run did not work correctly. This should be
~~~
        rotatelabels=False, *, normalize=None, data=None):
~~~

I tested locally, and boilerplate.py gave the correct output for me. Not sure what went wrong, but you can also do the change manually if you don't want to dig into this.